### PR TITLE
[ENHANCEMENT] Use absolute imports in tests

### PR DESCRIPTION
**Pull request by kamalaknn**
Continuation of #5203

### CODE REVIEW

**Comment by kamalaknn** [comment]
Not sure why Travis failed on node 0.10

```bash
 1305 passing (24m)
  9 pending
  1 failing
  1) update task do update "after each" hook for "says 'a new version is available' and asks you to confirm you want to update":
     Error: timeout of 5000ms exceeded. Ensure the done() callback is being called in this test.
```

Maybe a random timeout?

Travis is green on  node 0.12 and 4.2

**Comment by kellyselden** [comment]
@kamalaknn I restarted just in case

**Comment by kamalaknn** [comment]
Thanks @kellyselden. Travis looks green now :smiley:

**Comment by kellyselden** [comment]
I don't think `ember-cli-get-dependency-depth` is used anymore after this change. Can you confirm?

**Comment by kamalaknn** [comment]
@kellyselden Yes, `ember-cli-get-dependency-depth` is not used anymore. I've removed it from `package.json` along with this.

File: blueprints/helper-test/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 /*jshint node:true*/
 
-var getDependencyDepth = require('ember-cli-get-dependency-depth');
 var testInfo = require('ember-cli-test-info');
+var stringUtils  = require('ember-cli-string-utils');
```
Comment (kellyselden) [inline blueprints/helper-test/index.js]: extra space

File: blueprints/initializer-test/index.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
 /*jshint node:true*/
 
-var getDependencyDepth = require('ember-cli-get-dependency-depth');
 var testInfo = require('ember-cli-test-info');
+var stringUtils  = require('ember-cli-string-utils');
```
Comment (kellyselden) [inline blueprints/initializer-test/index.js]: extra space

File: blueprints/instance-initializer-test/index.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
 /*jshint node:true*/
 
-var getDependencyDepth = require('../../lib/utilities/get-dependency-depth');
 var testInfo = require('ember-cli-test-info');
+var stringUtils  = require('ember-cli-string-utils');
```
Comment (kellyselden) [inline blueprints/instance-initializer-test/index.js]: extra space

**Comment by stefanpenner** [comment]
@rwjblue r?

**Comment by rwjblue** [comment]
@homu r+

**Pull request merged**